Phaethon/InternalApi/InternalApi.csproj

```diff
@@ -161,11 +162,6 @@
       <Name>Core</Name>
     </ProjectReference>
   </ItemGroup>
-  <ItemGroup>
```

Comment:
I hope you correctly merged the migrations

Reply:
yes i did and everything works, idk what is that though, maybe previous merge was bad :/